internal/commands/push/command.go

```diff
@@ -266,31 +266,26 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return nil
 	}
 
-	if len(appDiffs) > 0 {
+	var draft realm.AppDraft
+	if len(appDiffs) > 0 || dependenciesDiffs.Len() > 0 || hostingDiffs.Size() > 0 {
```

Comment:
I just realized that at this point we already know that one of these is true since we'll return early on line 231 if this wasn't true. So there is no need for this if statement.